tests/functionality/scopflow/selfcheck.cpp

```diff
@@ -8,35 +8,39 @@ struct ScopflowFunctionalityTestParameters {
   MPI_Comm comm = MPI_COMM_WORLD;
 
   /* Parameters required to set up and test a SCOPFlow */
-  std::string solver = "";
-  std::string model = "";
+  std::string solver = SCOPFLOWOptions::solver.default_value;
+  std::string model = SCOPFLOWOptions::model.default_value;
   std::string network = "";
   std::string contingencies = "";
   std::string pload = "";
   std::string qload = "";
   std::string windgen = "";
   std::string description = "";
-  int num_contingencies;
-  double tolerance;
+  int num_contingencies = 0;
+  double tolerance = SCOPFLOWOptions::tolerance.default_value;
   double warning_tolerance = 0.01;
   int iter_range = 0;
-  double duration;
-  double dT;
-  int mode;
-  bool multiperiod;
+  double duration = SCOPFLOWOptions::duration.default_value;
+  double dT = SCOPFLOWOptions::dT.default_value;
+  int mode = SCOPFLOWOptions::mode.default_value;
+  bool multiperiod = SCOPFLOWOptions::enable_multiperiod.default_value;
 
   /* Parameters used to modify underlying opflow */
-  std::string opflow_initialization_string;
+  std::string opflow_initialization_string =
+      OPFLOWOptions::initialization.default_value;
   int opflow_initialization;
-  std::string opflow_genbusvoltage_string;
+  std::string opflow_genbusvoltage_string =
+      OPFLOWOptions::genbusvoltage.default_value;
   int opflow_genbusvoltage;
-  std::string subproblem_solver;
-  std::string subproblem_model;
-  bool enable_powerimbalance_variables;
-  bool ignore_lineflow_constraints;
-  int verbosity_level;
+  std::string subproblem_solver = OPFLOWOptions::solver.default_value;
```

Comment:
Again I question if these default initializations are actually necessary. Is there anything that motivated these changes? `develop` seems to have been chugging along just fine without it for a while